scaffolds/php/README.md

```diff
@@ -0,0 +1,46 @@
+# PHP Scaffold for Code Retreat
+
+This scaffold comes with severals tools that work with docker under the hood.
+- `php`: a php 7.1 executable
+- `phpspec`: a php test framework
+- `composer`: a php dependencies manager
+
+Run
+```bash
+docker.sh [command]
```

Comment:
You probably don't have the current directory `.` in your `PATH`, and it's good.
As consequence, isn't it `./docker.sh`?

Reply:
Good catch!